### ISSUE TITLE
Readressing DMX channels

### ISSUE THREAD

**Issue by Lesateq** [Marthaarman/dmx-transceiver#4]
I have started programming microcontrollers after  25 years  not having worked with C. So forgive me I am a bit rusty...  I would like to make a channel converter. Simple  and  cheap DMX controllers work with fixed DMX channels. So I would like to change the outgoing  channels of the DMX controller and send the adapted addresses to fixtures e.d. in the theater I work for. Changing the addresses of the fixtures in not easy/possible. It is quite a complex infrastructure... Would that be possible with the DMX-Transceiver lib? I experienced that the horsepower of the Arduino is limited,  DMX runs at 250K and 'talks' the whole time. Just like my daughters...

**Comment by Marthaarman** [comment]
> I have started programming microcontrollers after 25 years not having worked with C. So forgive me I am a bit rusty... I would like to make a channel converter. Simple and cheap DMX controllers work with fixed DMX channels. So I would like to change the outgoing channels of the DMX controller and send the adapted addresses to fixtures e.d. in the theater I work for. Changing the addresses of the fixtures in not easy/possible. It is quite a complex infrastructure... Would that be possible with the DMX-Transceiver lib? I experienced that the horsepower of the Arduino is limited, DMX runs at 250K and 'talks' the whole time. Just like my daughters...

Yes should be possible. I have an implementation myself for which I hook up a simple light switch to the arduino. Once pressed the incoming DMX channels are slightly altered and then transmitted. Let me know what I can do for you.
